### ISSUE TITLE
Improved detectors/sensors

### ISSUE THREAD

**Issue by DieFluoro** [studio-minus/ppg-bugs-and-requests#1963]
**Feature description**
Allow for the sensors to have a cone of vision. Either make a new variant or create it as a context menu option. similar to having the choice between double trigger / single trigger, id like the choice between detector beam and detector cone. Alternatively you could make it so there is only a cone with the option to edit the field of view in degrees such that if someone still prefers a beam they may set the field of view to very little.

**Feature purpose**
The niche this would fill that made me think of the idea was while im trying to make a custom sentry turret that needs to detect close targets while still maintaining range, i have to hook up multiple detectors so that there is a triangle of detection as opposed to a beam.

**Comment by DindinYT37** [comment]
So a detector but wider range?

**Comment by DieFluoro** [comment]
yes, a detector with a field of view instead of a linear range maybe even a proximity detector with a customizeable radius

**Comment by EEASAS** [comment]
good idea
